### DISCUSSION TITLE
CIP-2 Issue 20,000,000 $CXN tokens to Constellation Fund

### DISCUSSION THREAD

**Discussion by takcxn** [cxndao/constellation#42] [CIPs]
**Goal:**

Issue 20,000,000 million $CXN tokens to Constellation Fund (“**CXNF**” or “**Fund**”) to facilitate the general fundraising and 

**Single Point of Contact:**

Tak Lo

**Sponsors:**

maddog_xyz#7920, SonnyBeez #9569

**Description:**

**_Partnering with Constellation Fund_**

The Fund is an independent venture investment company created to support Constellation DAO (“CXND” or “DAO”). It is run by the Sponsors of this proposal who serve as the Investment Committee members and General Partners of the Fund. For reasons of compliance, the DAO is a separate, arms-length entity that provides services to the Fund in exchange for Carried Interest. This proposal establishes a partnership agreement between the DAO (and its future entities) and the Fund to fund the DAO’s treasury.

The Fund is planning to initiate fundraising in early Q3-2022. CXNF is launching as a one-of-a-kind VC fund that has a strategic partnership with Constellation DAO, which provides it a significant competitive advantage in relation to sourcing and all the steps in-between that and ultimately exit.

As part of the fundraising strategy, potential limited partners will be offered $CXN tokens as an additional return kicker (with potential to earn more $CXN tokens via DAO contributions), which we believe will drive potential additional engagement touchpoints for CXND and ultimately expand the overall network.     

Following fundraising, $CXN tokens will be required by CXNF to establish a working relationship with the Constellation DAO Venture Capital Team (“CXND VC Team”) for various tasks including sourcing, due diligence and ongoing portfolio management.

_**Constellation DAO Venture Capital Team**_
(The formation of the CXND VC Team is not part of the proposal as it is understood to be in development and its formation does not require a vote (but any request for $CXN tokens would); however briefly describing for purposes of Benefits (see below))

CXND VC Team will be open and inclusive to all members of Constellation DAO, regardless of level of experience and skill sets. Engagement between CXNF and CXND VC Team is expected to be high, with opportunities for members of the CXND VC Team to contribute their insights, skills and resources to any given opportunity in exchange for $CXN tokens.

(The $CXN tokens used will be contributed by both CXNF and CXND. Note that the current proposal is related solely to the issuance of $CXN tokens to CXNF. _A proposal for the aforementioned $CXN tokens in relation to CXND VC Team compensation will be voted on in the future) _

**Proposal:**   

In exchange for 20% of total $CXN tokens (or 20,000,000 $CXN tokens) to be distributed to CXNF (vesting schedule below), CXND will receive the Carried Interest (“CI”) of CXNF, which is defined as: 20.0% of the amount of return in excess of 2.0x contributed capital. Any taxes required to be paid by the CXNF GPs for transferring this amount will reduce the Carried Interest before it is sent to the DAO.

Vesting schedule for the $CXN tokens received by CXNF shall be 4,000,000 tokens per annum (with monthly vesting) for a period of five (5) years.

**Benefits to Constellation DAO**

The success of Constellation Fund provides significant benefits to Constellation DAO, including the following:
* Enhanced press and media coverage, allowing for increasing member growth
* CXND member engagement, allowing for member retention via aligned incentives
* Carried Interest provides a base level of value for $CXN token (see [table](https://docs.google.com/spreadsheets/d/1dv9IOGHcxVnZKRJ1O1OFOB3RAhXfmonp/edit#gid=626407166) below for sensitivity analysis)

Once approved by this DAO, the agreement is cancellable only by mutual agreement between the DAO (by proposal) and CXNF.

**Duration**:

Start: Upon approval
End: Five (5) years from the effective date of the CXNF (for clarity, CXNF has the ability to extend the fund like by up to two (2) years)

**Comment by mattjoyal** [comment]
Hey @takcxn this is looking great. 

Question regarding the $CXN governance tokens owned by the Fund – will the Fund have voting power within the DAO or are those tokens not considered eligible for voting matters within CXN DAO?

**Reply by SonnyBeez** [reply]
> 

Hi @mattjoyal the expectation is that the tokens issued would have voting rights

**Reply by jim-stanczak** [reply]
Just to be super clear, we are following Orange Dao's lead: "Unvested tokens are ineligible for use in voting."
=> Do we need to spell this out explicitly?

**Reply by mattjoyal** [reply]
Per our charter, 5.4 Quorum, 
5.4.3. Tokens held by the Treasury and unvested tokens do not carry any voting power and do not count towards quorum.

[Discord discussion continued here](https://discord.com/channels/920456009267429417/1013971941494898838/1019215305744076891)
